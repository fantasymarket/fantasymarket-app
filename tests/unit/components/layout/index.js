import React from 'react';
import { render } from '@testing-library/react';

import Layout from '@components/layout';

describe('Layout component', () => {
	const container = render(<Layout>Page Content </Layout>);

	it('renders the component', () => {
		expect(container.baseElement).toMatchSnapshot();
	});
});
