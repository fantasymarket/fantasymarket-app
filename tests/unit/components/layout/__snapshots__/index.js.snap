// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout component renders the component 1`] = `
<body>
  <div>
    <div
      class="sc-fzoLsD gTbaXh"
    >
      <div
        class="sc-Axmtr gSEeTk"
      >
        <a
          href="/"
        >
          <h1>
            FantasyMarket
          </h1>
        </a>
        <ul>
          <li>
            <a
              href="/dashboard"
            >
              dashboard
            </a>
          </li>
          <li>
            <a
              href="/portfolio"
            >
              portfolio
            </a>
          </li>
          <li>
            <a
              href="/orders"
            >
              orders
            </a>
          </li>
          <li>
            <a
              href="/help"
            >
              help
            </a>
          </li>
          <li>
            <a
              href="/settings"
            >
              settings
            </a>
          </li>
          <li
            class="undefined settings-btn"
          >
            <a
              href="/settings"
            >
              settings
            </a>
          </li>
        </ul>
        <div>
          <a
            class="settings-btn"
          >
            <button
              class="sc-AxirZ bLrhwI"
            >
              Login
            </button>
          </a>
        </div>
      </div>
      <div
        class="sc-fzpans lhAyIL"
      >
        Page Content 
      </div>
      <div
        class="sc-fzozJi gXBskI"
      >
        <h3>
          Â© FantasyMarket 2020
        </h3>
      </div>
    </div>
  </div>
</body>
`;
